readme.md

```diff
@@ -39,6 +39,16 @@ details.branchName // is null if the repository in detached HEAD mode
 details.isCleanTag
 ```
 
+You can optionally configure the plugin to search a subset of tags with `prefix`. Inclusion of the `prefix` in the returned version is toggled
+with `includePrefix` (default: `false`). Example configuration:
+
+```groovy
+gitVersionOpt {
+    prefix = "v"
```

Comment:
I can switch the example to what Lerna does, where it will create a tag for each package in the repo. So I could have:

Product@X.Y.Z
product-api@A.B.C
product-app@C.D.E

This allows you to easily navigate to the commit where a version was bumped for any package in the repository. This means that gitVersion() won't work because it assumes you are only tracking versions for one thing in the repository.